System.Compiler/Reader.cs

```diff
@@ -2129,13 +2129,28 @@ private TypeNode GetTypeFromTypeName(SerializedTypeName typeName)
         string lookupName = localInfo.Name;
         bool hasNoPDBName = String.IsNullOrEmpty(lookupName);
 #if !FxCop
-        string name = hasNoPDBName ? "local"+i : lookupName;
+        
+        // In some case it is very useful to know is the local has real name or
```

Comment:
:bulb: I find this comment unnecessary considering we clearly have an `Anonymous` property to assign below. I would prefer to use the single-line statement from the left side, and just use this below:

```csharp
loc.Anonymous = hosNoPDBName;
```